taxa/profile/tpimageeditor.php

```diff
@@ -30,6 +30,7 @@
 </head>
 <body>
 	<div id="innertext" style="background-color:white;">
+		<h1 class="page-heading"><?= $LANG['TP_IMAGE_EDITOR']; ?></h1>
```

Comment:
Make screen reader only and change to "Taxon Profile Image Editor"

Reply:
Fixed in 6740567ce.